src/java/org/apache/cassandra/config/Properties.java

```diff
@@ -68,6 +71,35 @@ public static Property andThen(Property root, Property leaf)
         return andThen(root, leaf, DELIMITER);
     }
 
+    @Nullable
+    public static Property andThenMap(Property root, String key, String delimiter)
```

Comment:
> I think we can remove all this if we simplify the config

how so?  its very useful to have per-verb overrides, so only way to avoid this is to have a field per verb rather than a map...

Reply:
following your feedback to drop verb, I removed this